tests/enable-gcp-services/idempotent-prune/services.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: blueprints.cloud.google.com/v1alpha1
+kind: ProjectServiceList
+metadata:
+  name: proj1-service
+  annotations:
+    config.kubernetes.io/local-config: "true"
+spec:
+  services: # We expect only generated service resources by proj1-service ProjectServiceList to be pruned and regenarated. All other service resources remain unmodified.
```

Comment:
I don't see this test exercises the prune behavior.
IIUC if I initially have a KCC object for `bigquery.googleapis.com` with annotation `blueprints.cloud.google.com/managed-by-enable-gcp-services: 'proj1-service'`, after I run the function, this bigquery KCC resource should disappear, right?

Reply:
Thats correct, I added this scenario to the test as well.